recipes/pypandoc/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.1.3" %}
+package:
+  name: pypandoc
+  version: {{ version }}
+
+source:
+  fn: pypandoc-{{ version }}.zip
+  url: https://github.com/bebraw/pypandoc/archive/v{{ version }}.zip
```

Comment:
jein: it is, but I don't want to take it from there as it currently contains an old pandoc download by accident and only the next release will be fixed...